packages/utils/arrays.ts

```diff
@@ -5,7 +5,7 @@ type Many<T> = T | ReadonlyArray<T>
 /** like `_.castArray`, except falsy value returns empty array. */
 export const castArray = <T>(arr: Many<T>): T[] => {
   if (!arr && (arr as any) !== 0) return []
-  return Array.isArray(arr) ? arr : [arr]
+  return Array.isArray(arr) ? (arr as T[]) : [arr as T]
```

Comment:
There seems to be no problem under the ts version that the code currently depends on, and it may not need to be modified.

Reply:
i learn from it